templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} new car model</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Submit">

```

Comment:
According to the checklist, you should have a 'Cancel' button in addition to the 'Submit' one. The 'Cancel' button should lead to the previous page the user was on. Consider adding it.